GoogleForCreators/web-stories-wp
# Media Modal: Fix invisible crop selection

## Context



## Summary

There is a issue with the imgareaselect library. Fix this by patching it locally and overriding and loading our version.  

## Relevant Technical Choices



## To-do



## User-facing changes



## Testing Instructions



### QA






This PR can be tested by following these steps:
1. Open editor
1. Click to edit/add poster image
1. Make browser window small
1. Select a large image that would need to be cropped
1. See cropping selection
1. Click "Crop image" 

Try resizing window to different sizing while in cropping screen.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #8398
